modules/service/alias.go

```diff
@@ -37,7 +31,7 @@ const (
 	RequestAccName          = types.RequestAccName
 	TaxAccName              = types.TaxAccName
 	MetricsSubsystem        = types.MetricsSubsystem
-	TypeMsgSvcDef           = types.TypeMsgSvcDef
+	TypeMsgDefineService    = types.TypeMsgDefineService
```

Comment:
Msg unified starts with the prefix "Msg"，such as  `TypeMsgDefineService  -> MsgDefineService `